criu/crtools.c

```diff
@@ -541,6 +541,12 @@ int main(int argc, char *argv[], char *envp[])
 	       "                        pages images of previous dump\n"
 	       "                        when used on restore, as soon as page is restored, it\n"
 	       "                        will be punched from the image\n"
+	       "  --zero-pages          don't dump pages containing only zero bytes. This is a\n"
```

Comment:
The name `--skip-zero-pages` might be more appropriate for this option.

Reply:
Agree.